PowerEditor/installer/nativeLang/english.xml

```diff
@@ -13,11 +13,11 @@ Translation note:
 					<Item menuId="file" name="&amp;File"/>
 					<Item menuId="edit" name="&amp;Edit"/>
 					<Item menuId="search" name="&amp;Search"/>
-					<Item menuId="view" name="&amp;View"/>
+					<Item menuId="view" name="&amp;Display"/><!-- Because the term View is also used in the Menu "Move to Other View" and "Clone to Other View" -->
 					<Item menuId="encoding" name="E&amp;ncoding"/>
 					<Item menuId="language" name="&amp;Language"/>
 					<Item menuId="settings" name="Se&amp;ttings"/>
-					<Item menuId="tools" name="To&amp;ols"/>
+					<Item menuId="tools" name="To&amp;ols"/> and
```

Comment:
It shouldn't be there.

Reply:
For this point it was an error on my part and I subsequently deleted it.

Reply:
> It shouldn't be there.

For this point it was an error on my part and I subsequently deleted it.